src/structurizr/workspace.py

```diff
@@ -195,18 +195,54 @@ def __init__(
 
     @classmethod
     def load(cls, filename: Union[str, Path]) -> "Workspace":
-        """"""
+        """Load a workspace from a file, which may be zipped."""
         filename = Path(filename)
         try:
             with gzip.open(filename, "rt") as handle:
-                ws_io = WorkspaceIO.parse_raw(handle.read())
+                return cls.loads(handle.read())
         except FileNotFoundError as error:
             raise error
         except OSError:
             with filename.open() as handle:
-                ws_io = WorkspaceIO.parse_raw(handle.read())
+                return cls.loads(handle.read())
+
+    @classmethod
+    def loads(cls, json_string: str) -> "Workspace":
+        """Load a workspace from a JSON string."""
+        ws_io = WorkspaceIO.parse_raw(json_string)
         return cls.hydrate(ws_io)
 
+    def dump(
+        self,
+        filename: Union[str, Path],
+        *,
+        zip: bool = False,
+        indent: Optional[int] = None,
+        **kwargs
+    ):
+        """
+        Save a workspace to a file, optionally zipped.
```

Comment:
I would strictly call it gzip everywhere so as not to confuse it with a zip archive.